plugins/suggested-blocks/README.md

```diff
@@ -0,0 +1,49 @@
+# @blockly/suggested-blocks [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+A [Blockly](https://www.npmjs.com/package/blockly) plugin that suggests blocks for the user based on which blocks they've used already in the workspace.
+
+## Installation
+
+### Yarn
+```
+yarn add @blockly/suggested-blocks
+```
+
+### npm
+```
+npm install @blockly/suggested-blocks --save
+```
+
+## Usage
+```js
+import * as Blockly from 'blockly';
+import {toolboxCategories} from '@blockly/dev-tools';
```

Comment:
@maribethb is this a recommended usecase for dev-tools? I think we might want to change the example to not use this.

On the other hand, the comment on `indexToInsert` might be good enough to discourage people from doing this if it's not intended.

Reply:
Resolving for now, can reopen if still concerned.